README.md

```
@@ -1,775 +1,302 @@
-
-### Automated Test Verification
+---
 
-Run comprehensive test suite before manual QA:
+## 🏗️ **Architecture Overview**
 
-```bash
-# Run all tests
-npm run test
+### **Frontend Architecture**
+- **React 19** + **TypeScript** + **Material-UI v5**
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)